osu.Game.Rulesets.Mania/Difficulty/Skills/README.md

```diff
@@ -0,0 +1,280 @@
+
+To better understand what are the possibilities, we illustrate all possible states.
+Impossible cases are marked with `X` as [notes are sorted](#rules)
+
+```
+Legend             E.g. 
++--------------+   +-------------+
+| (State Name) |   | (B2)        | Currently, 
+| Prev Note    |   | [====]      | we have a note (O) & a hold started before this note 
+| This Note    |   |      O      | then ended on the same time as us.
++--------------+   +-------------+
+
+       The column titles are with respect to the previous note.
+       E.g. B2: Long Note End is On the Head of the current Note 
+      +-------------+-------------+-------------+-------------+--------------+
+      | Before Head | On Head     | On Body     | On Tail     | After Tail   |
+      | (1)         | (2)         | (3)         | (4)         | (5)          |
++-----+-------------+-------------+-------------+-------------+--------------+
+| (A) | (A1)        | (A2)        |             |             |              |
+|     | O           |      O      |      X      |      X      |      X       |
+|     |      O      |      O      |             |             |              |
++-----+-------------+-------------+-------------+-------------+--------------+
+| (B) | (B1)        | (B2)        |             |             | (B5)         |
+|     | [==]        | [====]      |      X      |      X      | [==========] |
+|     |      O      |      O      |             |             |      O       |
++-----+-------------+-------------+-------------+-------------+--------------+
+| (C) |             |             |             |             | (C5)         |
+|     |      X      |      x      |      X      |      X      |      [===]   |
+|     |             |             |             |             |      O       |
++-----+-------------+-------------+-------------+-------------+--------------+
+| (D) | (D1)        | (D2)        |             |             |              |
+|     | O           |      O      |      X      |      X      |      X       |
+|     |      [===]  |      [===]  |             |             |              |
++-----+-------------+-------------+-------------+-------------+--------------+
+| (E) | (E1)        | (E2)        | (E3)        | (E4)        | (E5)         |
+|     | [==]        | [====]      | [======]    | [=========] | [==========] |
+|     |      [===]  |      [===]  |      [===]  |       [===] |      [===]   |
++-----+-------------+-------------+-------------+-------------+--------------+
+| (F) |             |             | (F3)        | (F4)        | (F5)         |
+|     |      X      |      X      |      [=]    |       [===] |      [=====] |
+|     |             |             |      [===]  |       [===] |      [===]   |
++-----+-------------+-------------+-------------+-------------+--------------+
+```
+
+In our script, we trigger strain bonuses under 2 conditions
+
+- Column 3: `endOnBodyBias`
+- Column 5: `endAfterTailWeight`
```

Comment:
The huge table above is presented and then only two columns of the huge table matter. Does the huge table then need to exist at all? Can't you just describe the interesting cases in those columns separately?